src/Hydra/Serializer/ErrorNormalizer.php

```diff
@@ -42,6 +43,10 @@ public function normalize($object, $format = null, array $context = [])
         $message = $object->getMessage();
         if ($this->debug) {
             $trace = $object->getTrace();
+        } else {
```

Comment:
We lose the mapping between exceptions and their HTTP status code (#797). What about passing [the correct HTTP status code](https://github.com/api-platform/core/blob/master/src/Action/ExceptionAction.php#L71) inside the serializer context?